torch/_subclasses/fake_impls.py

```diff
@@ -1022,11 +1023,17 @@ def slow(msg):
             for op in operands:
                 if not isinstance(op, torch.Tensor):
                     continue
-                is_contiguous = is_contiguous and op.is_contiguous(
-                    memory_format=torch.contiguous_format
+                is_contiguous = (
```

Comment:
this seems legit one way to verify that i have been doing is putting a PR where I always take the slow path and see what fails.

is_contiguous -> definitely_contiguous